src/makeChange.js

```diff
@@ -1,32 +1,52 @@
-const makeChange = ({price, amountGiven}) => {
-  let change = { quarters: 0, dimes: 0, nickels: 0, pennies: 0 },
-      remainder = amountGiven - price
-
-  const findChangeAmount = ( denomination, amount ) => {
-    let numberOfDenomination = Math.floor( remainder / amount )
-    change[denomination] = numberOfDenomination
-    remainder = remainder - ( numberOfDenomination * amount )
+const makeChange = ({ price, amountGiven, callback }) => {
```

Comment:
Here's another case where we are forcing users of this function to explicitly understand how this function works in order to be able to use it... Consider removing the injected callback, and invoking the function explicitly as part of the implementation.